hive_metastore_client/hive_mestastore_client.py

```diff
@@ -77,3 +83,75 @@ def add_columns_to_table(
 
         # call alter table to add columns
         self.alter_table(dbname=db_name, tbl_name=table_name, new_tbl=table)
+
+    def add_partitions_to_table(
+        self, db_name: str, table_name: str, partition_list: List[Partition]
+    ) -> None:
+        """
+        Add partitions to a table.
+
+        :param db_name: database name, where the table is at
+        :param table_name: table name, which the partitions belong to
+        :param partition_list: list of partitions to be added to the table
+        """
+        table = self.get_table(dbname=db_name, tbl_name=table_name)
+
+        partition_list_with_correct_location = self._format_partitions_location(
+            partition_list=partition_list,
+            table_storage_descriptor=table.sd,
+            table_partition_keys=table.partitionKeys,
+        )
+
+        # call method to add the formatted partitions
```

Comment:
I don't think this comment is pretty informative. I think it's a little redundant.